packages/runner-ct/src/app/RunnerCt.tsx

```diff
@@ -92,7 +92,9 @@ const RunnerCt = namedObserver('RunnerCt',
 
     const runSpec = React.useCallback((file: FileNode) => {
       setActiveIndex(0)
-      const selectedSpec = props.state.specs.find((spec) => spec.absolute.includes(file.relative))
+      // We request an absolute path from the dev server but the spec list displays relative paths
+      // For this reason to match the spec we remove leading relative paths. Eg ../../foo.js -> foo.js.
+      const selectedSpec = props.state.specs.find((spec) => spec.absolute.includes(file.relative.replace(/\.\.\//gi, '')))
```

Comment:
This regex `replace` is called on every item in the loop. Please assign it to a variable instead.

Reply:
I mean:
```
const filePath = file.relative.replace(/\.\.\//gi, '')
const selectedSpec = props.state.specs.find((spec) => spec.absolute.includes(filePath))
```

Reply:
🤦‍♂️ right.

Done.